### PR TITLE
Add OpenAI docs

### PR DESCRIPTION
**Pull request by colin-sentry**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [x] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

- https://github.com/getsentry/sentry-python/pull/2791
- https://github.com/getsentry/develop/pull/1183

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: docs/platforms/python/integrations/openai/index.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    dsn="___PUBLIC_DSN___",
+    enable_tracing=True,
+    traces_sample_rate=1.0,
+)
+
+client = OpenAI()
+```
+
+## Verify
+
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/openai/index.mdx:40]:
```suggestion

Verify that the integration works by inducing an error. The error and performance transaction should appear in your Sentry project. 
```

File: docs/platforms/python/integrations/openai/index.mdx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+It takes a couple of moments for the data to appear in [sentry.io](https://sentry.io).
+
+## Behavior
+
+- The OpenAI integration will connect Sentry with all supported OpenAI methods automatically.
+
+- All exceptions leading to an OpenAIException are reported.
+
+- The supported modules are currently `chat.completions.create` and `embeddings.create`
```
Comment (vivianyentran) [inline docs/platforms/python/integrations/openai/index.mdx:69]:
```suggestion
- The supported modules are currently `chat.completions.create` and `embeddings.create`.
```

**Review by vivianyentran — APPROVED** [review]
This doc looks great and is super easy to read! Thanks for adding this!

Before you merge, can you also make sure to add this integration to the [list of integrations](https://sentry-docs-git-openai.sentry.dev/platforms/python/integrations/#miscellaneous) (maybe under miscellaneous)?

**Comment by antonpirker** [comment]
Hey! I took the liberty to add the new OpenAI integration to the list of integrations. 

And also asked our designer to add the Logo to our platform icons.

**Pull request merged**